src/utils/SchemaReference.js

```diff
@@ -93,6 +94,7 @@ export function getEnumValueReference(typeInfo: any): EnumValueReference {
   return {
     kind: 'EnumValue',
     value: typeInfo.enumValue,
+    // $FlowFixMe
     type: getNamedType(typeInfo.inputType),
```

Comment:
Adding an invariant makes Flow happy:

![Screenshot 2019-05-29 09 09 15](https://user-images.githubusercontent.com/129910/58540546-7f0a6580-81f1-11e9-9514-a438539e4788.png)

But if I just change the entire function to `throw new Error("...")` all of the tests still pass, so clearly this method isn't tested. As such I'm worried that adding an invariant may cause breakage, so I think the `$FlowFixMe` is the best option for now 👍